contracts/interfaces/IRepaymentCalc.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.6.11;
 
-import { ICalc } from "../../../calculator/contracts/interfaces/ICalc.sol";
+import { ICalc } from "../../lib/calculator/contracts/interfaces/ICalc.sol";
```

Comment:
Were moving to use `modules` across the board instead of `lib`. This will also solve the src problem for ds-test.